packages/backend/src/open_payments/payment/outgoing/service.ts

```diff
@@ -208,7 +208,9 @@ function validatePaymentInterval({
 }): boolean {
   return (
     !limits.paymentInterval ||
-    (limits.paymentInterval.start.toMillis() <= payment.createdAt.getTime() &&
+    (limits.paymentInterval.start !== null &&
+      limits.paymentInterval.start.toMillis() <= payment.createdAt.getTime() &&
+      limits.paymentInterval.end !== null &&
```

Comment:
```suggestion
    (limits.paymentInterval.start != null &&
      limits.paymentInterval.start.toMillis() <= payment.createdAt.getTime() &&
      limits.paymentInterval.end != null &&
```
Maybe to encapsulate `undefined`s as well just in case?